### ISSUE TITLE
Should Daily Notes core plug-in be disabled?

### ISSUE THREAD

**Issue by pm64** [liamcain/obsidian-periodic-notes#138]
Sorry for asking a dumb question, but after a bit of research I'm still not clear on whether the user is expected to disable the Daily Notes core plug-in when using obsidian-periodic-notes?

**Comment by anantshri** [comment]
this superseeds the daily notes you should be able to turn it off and switch to using this icon for daily notes.

**Comment by pm64** [comment]
Thank you @anantshri for the explanation. To be clear, are you saying that other plugins that depend on the core Daily Notes plugin (for example, the Calendar community plugin) are expected to work with no issues when obsidian-periodic-notes is enabled, and the core Daily Notes plugin is disabled?

**Comment by anantshri** [comment]
Yes correct. calendar and periodic notes are replacements of daily notes. periodic note is a direct 1:1 replacement with sidebar icon also.

**Comment by pm64** [comment]
Understood, thanks again!

**Comment by ksdavidc** [comment]
Just to be clear, MUST one disable the daily notes, or can one use both concurrently?

**Comment by danieltomasz** [comment]
You could use both, but it might be confusing (but I can have 2 sets of daily notes), if you define different paths for daily notes I dont know which one will be picked by calendar, et cetera
